buddyboss/buddyboss-platform
# Fix - Message Block member popup and member listing UI changes

### All Submissions:


Fixes #2875 

### How to test the changes in this Pull Request:

1. Create thread message with multiple users
2. Check Load more button CSS in message header
3. After that, click on Block a member 
![image](https://user-images.githubusercontent.com/30429982/132662810-2b51f67e-cba3-4b79-a8c7-a81a9e91cebb.png)
4. Click on Block button
![image](https://user-images.githubusercontent.com/30429982/132662923-63b69604-0a22-4e33-8468-8ebca050675e.png)
5. That blocked user should not be display in the popup( Block a member list in the message screen ( in 4 point screen ) ).

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crQQQgVQmKB

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Dependent on App Native
https://github.com/buddyboss/buddyboss-app-native/issues/1606
